media/css/mozorg/contribute/taskview.less

```diff
@@ -435,3 +403,64 @@ a[role="button"] {
 .hidden {
     display: none;
 }
+
+/* newsletter module styles */
+.newsletter-module {
```

Comment:
Ah yes, I moved the code into the main `taskview.less` and forgot to delete that one. Nice catch, thanks!